src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+
+@Entity
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @OneToOne
+    @JoinColumn(name = "country_id")
```

Comment:
The `@OneToOne` annotation is used here, which implies a one-to-one relationship with the `Country` entity. Ensure that this relationship is correctly mapped in the `Country` class if bidirectional mapping is required.